software/spmd/bsg_cuda_lite_runtime/float_matrix_mul_shared_mem/kernel_float_matrix_mul_shared_mem.c

```diff
@@ -99,6 +99,8 @@ void __attribute__ ((noinline)) subblock_shmem_matrix_mul_xposed (float *sh_A, f
 int  __attribute__ ((noinline)) kernel_float_matrix_mul_shared_mem(float *A, float *B, float *C, int M, int N, int P, int block_size_y, int block_size_x) {
 
 	
+	bsg_cuda_print_stat_start(0);
+
```

Comment:
Did you mean to include these?

Reply:
no just didn't want to add them every time to test. Removed now. But it wouldn't be the worst thing to have on them by default, would it?

Reply:
No, but it would be an overhead in regression. If we have them in all kernels, I would request that we have an on/off flag in the build flow.